### PR TITLE
Exemplar how to download other instructLab curated models like granite and use as default

### LINKED ISSUE #1282: Download instructions needed under `ilab download` for non-default, curated models (e.g., granite)
**Is your feature request related to a problem? Please describe.**
The taxonomy meetings have had questions/confusion on how to download/use granite models in place of the default model coded into ilab.  Primarily they reference the granite model(s) (e.g., ) referenced and published by ilab itself (i.e., https://huggingface.co/instructlab)

**Describe the solution you'd like**
A simple example on how to download the the granite model and place it into the `models` directory (by convention) would be a huge help to users who want to start using non-default models with granite being a key one we reference repeatedly.  

This example would provide the pattern (plug-and-play) for other models the project intends to adopt/publish and subsequently be used as the "default" for other commands. That is, the example should include additional help/refs. etc. to show how `ilab init` can (after download) be used to ref. the model as the new default.

**Additional context**
The goal of doc. changes is intended to be limited to the use case described above (i.e., to apply to instructLab curated models and not be exhaustive in describing how to download other models which may require tokens or bring in other complexities or entanglements.

### PR DESCRIPTION
**Pull request by mrutkows**
# Changes

During a recent taxonomy meeting, some attendees wanted to use the `granite` model (non-default, but curated/supported by instructLab) and were confused on how to download it and make it avail. to the other commands they used (in order to test new, prospective Q&A skills).  As this seemed to be a common concern for attendees, I promised to provide a PR to add this info. to the docs. to hopefully help them and others via a docs. PR.

**Which issue is resolved by this Pull Request:**
Resolves #1282 

**Description of your changes:**

Added and explained how one would download another supported model, i.e., "granite", into the `models` subdir and subsequently use `ilab init` to make it the default by reference.

**Note** This is a second attempt at the PR I closed in order to account for merge conflicts and have a clean review/discussion (see https://github.com/instructlab/instructlab/pull/1232).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -329,17 +329,29 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
   > **NOTE** ⏳ This command can take few minutes or immediately depending on your internet connection or model is cached. If you have issues connecting to Hugging Face, refer to the [Hugging Face discussion forum](https://discuss.huggingface.co/) for more details.
 
+  #### Downloading another instructLab model from Hugging Face
+
+  If you wish to download another [instructlab model](https://huggingface.co/instructlab) from Hugging Face, for example the [instructlab/granite-7b-lab-GGUF](https://huggingface.co/instructlab/granite-7b-lab-GGUF) model, you can  use this command with additional parameters as follows:
```
Comment (russellb) [inline README.md:334]:
```suggestion
  If you wish to download another [instructlab model](https://huggingface.co/instructlab) from Hugging Face, for example the [instructlab/granite-7b-lab-GGUF](https://huggingface.co/instructlab/granite-7b-lab-GGUF) model, you can use this command with additional parameters as follows:
```

**Review by russellb — APPROVED** [review]

**Review by leseb — CHANGES_REQUESTED** [review]
Thanks for clarifying the doc! Could you please update your commit message so that it matches the content of the PR description? Thanks!

File: README.md (hunk 1 above)
Comment (bjhargrave) [inline README.md:334]:
```suggestion
  If you wish to download another [InstructLab model](https://huggingface.co/instructlab) from Hugging Face, for example the [`instructlab/granite-7b-lab-GGUF`](https://huggingface.co/instructlab/granite-7b-lab-GGUF) model, you can  use this command with additional parameters as follows:
```

Comment (mrutkows) [reply README.md]: @bjhargrave Thanks BJ, like the suggestion.

**Comment by mrutkows** [comment]
> Thanks for clarifying the doc! Could you please update your commit message so that it matches the content of the PR description? Thanks!

AFAIK, the default on a squash/merge is to adopt the PR title (i.e., https://github.blog/changelog/2022-05-11-default-to-pr-titles-for-squash-merge-commit-messages/) and have never been asked to change commit descriptions and would have to ask for how to actually do such if my belief is correct.

**Comment by leseb** [comment]
> > Thanks for clarifying the doc! Could you please update your commit message so that it matches the content of the PR description? Thanks!
> 
> AFAIK, the default on a squash/merge (to main) is to adopt the PR title (i.e., https://github.blog/changelog/2022-05-11-default-to-pr-titles-for-squash-merge-commit-messages/) and have never been asked to change commit descriptions in a PR (and would have to ask for how to actually do such if my belief is correct).

We don't do squash merging. Our Mergify bot merges the PR as-is. You can read more about it [here](https://github.com/instructlab/dev-docs/blob/main/docs/github-merge-strategy.md#method-for-merging-a-pull-request).
Also, my request was not about squashing (which you should do before we merge) but to update the [initial commit](https://github.com/instructlab/instructlab/pull/1284/commits/b0ecd1d6c7e684dbcff14ea4b8a34f67f1a9d257) message (currently empty) with the description you added in the PR. Thanks!

**Review by bjhargrave — CHANGES_REQUESTED** [review]
LGTM but you need to squash (thus removing merge commits).

**Comment by mrutkows** [comment]
Closed in favor of PR https://github.com/instructlab/instructlab/pull/1334

**Comment by russellb** [comment]
> Closed in favor of PR #1334

You should have been able to just force push your updated changes to the same branch instead of opening a new PR, just FYI

**Pull request closed without merging**